README.md

```diff
@@ -16,35 +16,32 @@ To build the application, you need to install [Yarn](https://yarnpkg.com/) and r
 #   compile .scss files to .css:
 yarn
 
-# Each 'yarn add' to add new dependencies
-# will also rebuild the css from the
-# .scss files and throw an error
-# related to node-sass-chokidar,
-# which can be ignored.
-
 # Run unit tests in watch mode
 yarn test
 
 # Run unit tests with a code coverage report
-yarn run test:coverage
+yarn test:coverage
 
 # start the app running in the
 #   webpack development server:
 yarn start
 
 # start the app running in https mode
-# (needed for browser geolocation):
-yarn run start:https
+#   (needed for browser geolocation):
+yarn start:https
 
 # build the app into build folder
 #  for server deployment:
 yarn build
 
 # any updates to .scss files need
 #  to be compiled to css using:
-yarn build-css
+yarn clean-build-css
```

Comment:
Can we run the clean css command before every build as well to get rid of those duplicate css errors when just running straight up `yarn`?